src/utils/DragAndDrop.js

```diff
@@ -191,13 +195,21 @@ define(function (require, exports, module) {
             event.preventDefault();
 
             options.ondragover(event);
+            // adding functionality when dragging files on FILE bar
+            $("#sidebar").on("dragover", function() {
```

Comment:
Same thing here, should happen only once on initialization, not every time we handle the event.